frog/imports/client/TeacherView/utils/exportComponent.js

```diff
@@ -107,31 +111,44 @@ const objectProductPromise = id =>
     Meteor.call('get.object.product', id, (_, result) => resolve(result))
   );
 
-export const exportSession = (sessionId: string) => {
+const fetchSessionLIs = sessionId => {
```

Comment:
@lfaucon I'm not sure if absolutely all LIs get the correct sessionId - for example LIs that are part of the graph (templates), or LIs that are generated by operators (twitter etc). It might be, but we would have to check. Just FYI